### PR TITLE
Fix tags widget

### PR DESCRIPTION
**Pull request by sneridagh**
(Do not have an open vocabulary, and fix Volto widget).

### CODE REVIEW

File: src/collective/blog/behaviors/tags.py [hunk 1]
```diff
@@ -18,11 +19,16 @@ class IBlogTags(model.Schema):
     )
     blog_tags = schema.Tuple(
         title=_("label_blog_tags", default="Tags"),
-        value_type=schema.TextLine(),
+        value_type=Choice(source="collective.blog.tags"),
```
Comment (tisto) [inline src/collective/blog/behaviors/tags.py:22]: Don't we need an upgrade step here? Just wondering...

Comment (sneridagh) [reply src/collective/blog/behaviors/tags.py:22]: @tisto mmm why? In the case that someone has set a tag outside the restricted ones? In that case, which would be the procedure? Delete the offending tag? 

Maybe I am missing something else, aside of this... @davisagli ?

Comment (davisagli) [reply src/collective/blog/behaviors/tags.py:22]: As long as the values of the vocabulary are still text, it should not need an upgrade step. I'll test it with our project before approving

**Review by davisagli — APPROVED** [review]
Tested, looks good in both classic and volto. No upgrade needed.

File: src/collective/blog/behaviors/tags.py (hunk 1 above)
Comment (tisto) [reply src/collective/blog/behaviors/tags.py:22]: I am not a programmer any longer. Just asking stupid questions. ;)

Comment (davisagli) [reply src/collective/blog/behaviors/tags.py:22]: @tisto It wasn't a stupid question. It was worth the time to test and make sure.

**Pull request merged**